app/lib/backend/model_extensions/cinstance.rb

```diff
@@ -50,28 +45,6 @@ def delete_backend_application
         true
       end
 
-      def update_backend_user_key_to_application_id_mapping
```

Comment:
So this method used the `user_key`, deleting the application in backend and recreating it

We should not remove it but change its content using the correct API or is it replaced by `update_backend_application`?

Reply:
same as https://github.com/3scale/porta/pull/1843/files#r419911421
We should just use `update_backend_application` and `update_provider_backend_service_if_user_key_changed` instead, which is what we are doing